src/clj/puppetlabs/services/legacy_routes/legacy_routes_core.clj

```diff
@@ -0,0 +1,144 @@
+(ns puppetlabs.services.legacy-routes.legacy-routes-core
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [compojure.core :as compojure]
+            [ring.util.codec :as ring-codec]
+            [compojure.route :as route]
+            [clojure.tools.logging :as log]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; v3 => v4 header translation support functions
+
+; TODO: Improve the naive parsing here with a real library like
+; https://github.com/ToBeReplaced/http-accept-headers
```

Comment:
Good question.  I'd be tempted to just reference it directly to copying it but I haven't kept up on the conversations about removing liberator entirely.  I'd been thinking that was a soft goal and only in the sense of not automatically defaulting to layering new routes on top of Compojure/Bidi with it by default.  @cprice404 or @KevinCorcoran - do you think we'd we be going down a bad road by referencing liberator for a utility function here?